bigtable/google/cloud/bigtable/instance.py

```diff
@@ -508,27 +508,40 @@ def set_iam_policy(self, policy):
         """Sets the access control policy on an instance resource. Replaces any
         existing policy.
 
+        A Policy consists of a list of bindings. A binding binds a list of
+        members to a role, where the members can be user accounts, Google
+        groups, Google domains, and service accounts. A role is a named list
+        of permissions defined by IAM.
+        For more information about predefined roles currently supoprted
+        by Bigtable Instance please see
+        `Predefined roles
+        <https://cloud.google.com/bigtable/docs/access-control#roles>`_.
+        For more information about custom roles please see
+        `Custom roles
+        <https://cloud.google.com/bigtable/docs/access-control#custom-roles>`_.
+
```

Comment:
This kind of language belongs in the class docstring for `Policy`, and can just be refererenced from here.